data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1235 @@
+			choice
+				`	(Do nothing.)`
+					goto "home"
+				`	(Place your hand on your holster.)`
+			`	If the Gegno have an image of Earth, who knows what they know about you, or humanity in general. Slowly moving your hand to your holstered weapon, which might be rather obvious, you prepare yourself for any possible aggression. Even though you do so, the Gegno doesn't stop or hesitate to continue heading toward you, although some Gegno in the crowd seem slightly concerned.`
+
+			label "home"
+			`	Now standing not even two meters in front of you, the Gegno stares at you intently. An image of the world you're on right now slides over the one of Earth, and the Gegno gestures to it. "Gegno Dueyu Eitch echre, echre isend Gegno." The image shifts back to Earth, with the Gegno still gesturing to the screen. "Eyuu dueyu nnepta, viehetch eyuu." The Gegno now motions at the ground at your feet. "Viehetch eyuu ayyr dueyu Eitch. Udiyh tei tsider?"`
+			choice
+				`	(Don't say anything.)`
+				`	"...Dueyu Earth?"`
+					goto "dueyu"
+
+			`	Without saying a word, you continue to look at the Gegno, and it continues to stare back, pondering your silence. They make a snorting sound with their nose, then leaves you to join the other two behind them.`
+				goto "decision"
+
+			label "dueyu"
+			`	The Gegno leans closer to your height and shows an expression of possibly disapproval. You panick a little, and some of the others seem surprised that they got this close to you. They then speak in a low rumble, kind of like a whisper spoken aloud. "Eyuu idu keiou ghynem uf, "Dueyu"." Lifting back up, the Gegno heads back to the other two behind them.`
+
+			branch "done with the gegno"
+				or
+					or
+						"reputation: Gegno Vi" <= -100
+						"reputation: Gegno Scin" <= -50
+					and
+						"reputation: Gegno Vi" < 0
+						"reputation: Gegno Scin" < 0
+						has "zydee interference"
+					and
+						has "Full Vi Backlash: offered"
+						has "dead augen"
+					and
+						"gegno: encounters" >= 6
+						has "gegno: no proclamation"
+
+
+			label "decision"
+			action
+				"reputation: Gegno" = 1
+			`	For one last time, the trio beckons to the crowd followed by a long speech. An image of a space station appears on the screen, and a large discussion begins between the other Gegno, whom have formed back up with each other. Some arguments between various Gegno in the audience start happening, meanwhile, your Gegno guide bumps into you and starts to direct you away from the discussion, back to the cage lift with nobody else batting an eye.`
```

Comment:
Not fond of the first line. The way it is currently is to evoke more of a finality. Once again just makes it sound less important.

Reply:
The reason I suggested `Once again` is that narratively, it felt like you were looking into the future. At the time that the reader is processing that sentence, they shouldn't know that this is the last time. Since you're using present tense to describe what is currently happening to the player rather than past tense to describe what has already happened, the wording seems a little premature.

You could avoid that issue with something like this maybe
```suggestion
			`	With a sense of finality, the trio once again beckons to the crowd and begins another long speech. An image of a space station appears on the screen, and a large discussion begins between the other Gegno, whom have formed back up with each other. Some arguments between various Gegno in the audience start happening, meanwhile, your Gegno guide bumps into you and starts to direct you away from the discussion, back to the cage lift with nobody else batting an eye.`
```
This way, the trio is evoking a sense of finality with their actions rather than through the narration.

Reply:
Adjusted these a little bit but otherwise 👍